signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import argparse
+from pathlib import Path
+
+import wget
+import os
+from pose_format.pose import Pose
+from tqdm import tqdm
+import numpy as np
+import pympi
+
+from signwriting_transcription.pose_to_signwriting.data.preprocessing import preprocess
+from signwriting_transcription.pose_to_signwriting.data.pose_data_utils import build_pose_vocab
+from signwriting_transcription.pose_to_signwriting.data.datasets_pose import pose_to_matrix
+from signwriting_transcription.pose_to_signwriting.data.config import create_test_config
+from signwriting_transcription.pose_to_signwriting.joeynmt_pose.prediction import translate
+
+URL_HUGGINGFACE = "https://huggingface.co/spaces/ohadlanger/signwriting_transcription/resolve/main/"
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--model', required=True, type=str,
+                        help='model to use')
+    parser.add_argument('--pose', required=True, type=str, help='path to input pose file')
+    parser.add_argument('--elan', required=True, type=str, help='path to elan file')
+
+    return parser.parse_args()
+
+
+def main():
+    args = get_args()
+    print('Downloading model...')
+    if not os.path.exists("experiment"):
+        os.makedirs("experiment")
+    wget.download(f'{URL_HUGGINGFACE}{args.model}?download=true', 'experiment/best.ckpt')
```

Comment:
The best way to download files from huggingface is:
```py
from huggingface_hub import hf_hub_download
model_path = hf_hub_download(repo_id=HUGGINGFACE_REPO_ID, filename=args.model)
```
Then you can either create a symlink to this file, or copy it, your call.